elastic/beats
# Add more ECS fields to logs

## What does this PR do?

Adds service.name and event.dataset fields to ECS logs

## Checklist

~~- [ ] I have commented my code, particularly in hard-to-understand areas~~
~~- [ ] I have made corresponding changes to the documentation~~
~~- [ ] I have made corresponding change to the default configuration files~~
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~
~~- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.~~


## How to test this PR locally

Start a beat with default settings, both with and without `-e`, and see the implemented fields in the logs.

## Related issues

Closes #25997
